tests/CoreCLR/runtest/src/Common/CoreCLRTestLibrary/Logging.cs

```diff
@@ -0,0 +1,131 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+using System;
+using System.IO;
+using System.Security;
+using System.Text;
+
+namespace TestLibrary
```

Comment:
The CoreCLRTestLibrary should be part of the test drop. I do not think we want to have it duplicated in CoreRT.